### ISSUE TITLE
Peer2Profit Issue, not pulling "Access Denied"

### ISSUE THREAD

**Issue by Artiath** [OlivierGaland/CashFactory#49]
<img width="960" alt="image" src="https://github.com/OlivierGaland/CashFactory/assets/88950256/0c724041-4adb-4154-9752-7ede740d8224">

I'm going to try the stack without Peer2Profit, but this seems to be screwing it up for me.

OS: Ubuntu 20.04
Microcomputer: Orange Pi Zero 2
Architecture: arm64

**Comment by Artiath** [comment]
I found the closed issue with the exact same problem. I changed the docker-compose file to the updated one and it worked.

I'm keeping this open as it should be updated here so others do not have the same issue.

**Comment by furiaceka87** [comment]
Even if I modified the docker-compose I received an error:

```
peer2profit_1     | 2023-08-09T08:31:58Z | Getting config is error. NetworkError: 299 : Error transferring *** - server replied: Locked
peer2profit_1     | 2023-08-09T08:31:58Z | Error while getting of configuration. Repeat.

```

**Comment by RocketDevil** [comment]
If I see that correctly the "docker" version no longer receives any Data, All docker-powered instances of me all got under 200kb data transfer in the last 30 days...
So i guess they only want their pure "Linux" install for now to work, idk...

**Comment by trgcyln** [comment]
They banned docker for avoid abuse.

**Comment by mplok1** [comment]
It seems that the site/dashboard is no longer accessible to..
Is Peer2Profit always alive ?

**Comment by OlivierGaland** [comment]
Issue is fixed and provided in release v1.7
